websocket/handler_test.go

```diff
@@ -44,20 +45,20 @@ func TestPingHandler(t *testing.T) {
 
 func TestHandler_HandlerWSEvents(t *testing.T) {
 	// ---- Setup ----
-	hlr := &Handler{
-		websocketUpgrader: websocket.Upgrader{
-			ReadBufferSize:  10240,
-			WriteBufferSize: 10240,
-			CheckOrigin: func(r *http.Request) bool {
-				return true
-			},
-		},
-		user:              NewUserStore(2),
-		bufferChannel:     make(chan EventsBatch, 10),
+	upgrader := connection.NewUpgrader(connection.UpgraderConfig{
+		ReadBufferSize:    10240,
```

Comment:
We can make these values configurable or maybe a constant.

Reply:
As mentioned above, it's configurable.